### PR TITLE
chore: upgrade to node v16

### PR DESCRIPTION
**Pull request by baruchiro**
I always get an error `Promise.any is not a function`. Sometimes I get it only in debugging mode in VSCode.

I suspect it is supported only in the latest v14 versions, and I think we should move to v16 anyway.

### CODE REVIEW

File: .github/workflows/nodeCI.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [14.x, 16.x]
```
Comment (esakal) [inline .github/workflows/nodeCI.yml:11]: can you remove 14.x? otherwise we will continue to check it against 14 in the CI

Comment (baruchiro) [reply .github/workflows/nodeCI.yml:11]: Sure

Comment (baruchiro) [reply .github/workflows/nodeCI.yml]: Done

**Comment by eshaham** [comment]
@esakal done
@baruchiro would be helpful to update the Node version specified in the README.
It currently says Node 10 🤪 
Updating that would also retrigger the checks.

**Comment by baruchiro** [comment]
@eshaham README is updated
Please note you need to change the required checks in order to make this PR able to pass.
image

**Comment by eshaham** [comment]
I'll merge it forcefully, then.

**Pull request merged**